recipes/tarball.rb

```diff
@@ -177,7 +177,7 @@
   only_if { ::File.readlines('/etc/pam.d/su').grep(/# #{pam_limits}/).any? }
 end
 
-if node['cassandra']['use_systemd'] == false
+if node['cassandra']['use_initd'] == true
```

Comment:
An explicit comparison with `true` is excessive and looks weird in Ruby. Any value other than `false` or `nil` evaluates to `true`.